apps/theming/css/default.css

```
@@ -50,7 +50,7 @@
   --font-face: system-ui, -apple-system, 'Segoe UI', Roboto, Oxygen-Sans, Cantarell, Ubuntu, 'Helvetica Neue', 'Noto Sans', 'Liberation Sans', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
   --default-font-size: 15px;
   /* 1.5 x font-size for accessibility */
-  --default-line-height: 24px;
+  --default-line-height: 1.5em;
```

Setting `em` as line-height is not correct. This is an absolute value which means "1.5 * this element font-size". That means if you have 1.5em on a 15px parent, then all the children have 1.5 * 15 = 22.5px line height. Not matter what font-size they have, it's exactly 22.5px even for 100px and 10px font-size.

Only using unitless multiplier means "x1.5 of font-size", so children with 100px font size will have 150px line height, and 10px font-size will have 15px line height.

```suggestion
  --default-line-height: 1.5;
```

---

Example: https://codepen.io/ShGKme/pen/abgpYKB